a.go

```diff
@@ -0,0 +1,8 @@
+func a() {
```

Comment:
I don't think we want such a test file at the toplevel.

Reply:
oops, yeah good point